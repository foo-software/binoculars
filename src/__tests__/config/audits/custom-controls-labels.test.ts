import auditCustomControlsLabels from '../../../config/audits/custom-controls-labels';

describe('custom-controls-labels audit', () => {
  it('should match snapshot', () => {
    expect(auditCustomControlsLabels('en')).toMatchSnapshot();
  });
});
