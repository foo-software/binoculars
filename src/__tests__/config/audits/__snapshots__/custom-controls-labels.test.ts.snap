// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`custom-controls-labels audit should match snapshot 1`] = `
Object {
  "description": "Custom interactive controls have associated labels, provided by aria-label or aria-labelledby. [Learn more](https://web.dev/custom-controls-labels/).",
  "title": "Custom controls have associated labels",
}
`;
